src/EventListener/EasyAdminEventListener.php

```diff
@@ -44,7 +48,8 @@ public function onPostUpdate(GenericEvent $event): void
         $entity = $event->getSubject();
         if ($entity instanceof Submit) {
             if ($entity->hasStatusChanged()) {
-                $this->eventDispatcher->dispatch(new SubmitStatusChangedEvent($entity));
+                $this->eventDispatcher->dispatch(new SlackSubmitStatusChanged($entity));
+                $this->eventDispatcher->dispatch(new NotificationSubmitStatusChangedEvent($entity));
```

Comment:
This is wrong :) 

**One event, many listeners**

`SlackSubmitStatusChanged` this name was wrong in the first place. It should have been `SubmitUpdatedEvent`

Reply:
Well it is in reality `SubmitStatusChanged`, but since I had two of those in the same EventListener, I had to make aliases :).

But sure the listeners are quite messy, we could clean this up.

Reply:
The alias is really a bad idea :) the intent of listener should not leak in the event.
The event is generic